tests/model_selection/test_hyperband.py

```diff
@@ -0,0 +1,247 @@
+import numpy as np
+import scipy.stats
+from sklearn.linear_model import SGDClassifier
+
+import dask
+import dask.array as da
+from dask.distributed import Client, wait
+from distributed.utils_test import loop, cluster, gen_cluster  # noqa: F401
+from tornado import gen
+
+from dask_ml.datasets import make_classification
+from dask_ml.model_selection import HyperbandCV
+from dask_ml.wrappers import Incremental
+from dask_ml.model_selection._hyperband import _partial_fit
+from dask_ml.utils import ConstantFunction
+from distributed.metrics import time
+
+import pytest
+
+
+@pytest.mark.parametrize("array_type,library",  # noqa: F811
+                         [("dask.array", "dask-ml"),
+                          ("numpy", "sklearn"),
+                          ("numpy", "test")])
+def test_sklearn(array_type, library, loop, max_iter=27):
+    with cluster() as (s, [a, b]):
+        with Client(s['address'], loop=loop):
+            chunk_size = 100  # make dask array with one chunk
```

Comment:
I'm surprised that we only test with only a single chunk.  Is there a reason for this?